segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorage.java

```diff
@@ -261,7 +261,8 @@ public void initialize(long containerEpoch) {
                                     // Adjust its length;
                                     if (chunkInfo.getLength() != lastChunk.getLength()) {
                                         Preconditions.checkState(chunkInfo.getLength() > lastChunk.getLength(),
-                                                "Length of last chunk on LTS must be greater than what is in metadata.");
+                                                "Length of last chunk on LTS must be greater than what is in metadata. Chunk={} length={}",
```

Comment:
Please fix the string format. Use `%s` instead of `{}`.

Reply:
fixed